tests/Benchmarks/Benchmarks.csproj

```diff
@@ -31,6 +31,7 @@
     <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="$(BenchmarkDotNetVersion)" />
     <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="2.0.30" />
     <PackageReference Include="System.Text.Encodings.Web" Version="$(CoreFxStableVersion)" />
+    <PackageReference Include="xunit" Version="$(XUnitVersion)" IsImplicitlyDefined="true" PrivateAssets="all" Publish="true"/>
```

Comment:
Can you please explain what these addition properties do and why they are needed (`IsImplicitlyDefined="true" PrivateAssets="all" Publish="true"`)?

Reply:
`IsImpicitlyDefined` is so that you can't change the version in the editor. `Publish=true` tells the SDK to include it in the publish folder because we're marking all the assets as `Private`. `PrivateAssets=all` is to tell the SDK to not flow this dependency as a transitive dependency to projects that take a dependency on this project or package.

Reply:
Thanks for the explanation, @safern 

> `IsImpicitlyDefined` is so that you can't change the version in the editor.

And why do we want to disallow that, for the xunit package reference, specifically?

Reply:
We usually do that for our tooling specially since it is a shared version across different projects. This is using `XunitVersion` msbuild property. I bet that if we allowed that, VS would change `XUnitVersion` for the actual textual version, so then this project wouldn't use the shared property.